class_generator/class_generator.py

```diff
@@ -337,31 +367,14 @@ def get_arg_params(
     name = convert_camel_case_to_snake_case(string_=_orig_name)
     type_ = _type.strip()
     required: bool = "-required-" in splited_field
-    type_from_dict: str = TYPE_MAPPING.get(type_, "Dict[Any, Any]")
-    type_from_dict_for_init = type_from_dict
-
-    # All fields must be set with Optional since resource can have yaml_file to cover all args.
-    if type_from_dict == "Dict[Any, Any]":
-        type_from_dict_for_init = "Optional[Dict[str, Any]] = None"
-
-    if type_from_dict == "List[Any]":
-        type_from_dict_for_init = "Optional[List[Any]] = None"
-
-    if type_from_dict == "str":
-        type_from_dict_for_init = 'Optional[str] = ""'
-
-    if type_from_dict == "bool":
-        type_from_dict_for_init = "Optional[bool] = None"
-
-    if type_from_dict == "int":
-        type_from_dict_for_init = "Optional[int] = None"
+    type_from_dict: Dict[str, str] = TYPE_MAPPING.get(type_, TYPE_MAPPING["<Object>"])
 
     _res: Dict[str, Any] = {
         "name-from-explain": _orig_name,
         "name-for-class-arg": name,
-        "type-for-class-arg": f"{name}: {type_from_dict_for_init}",
+        "type-for-class-arg": f"{name}: {type_from_dict['type_for_init']}",
```

Comment:
why not just get it from the old dict that we had?

Reply:
now we have one dict which holds both type and type_for_init; i think it is easier to follow